liquidsoap.spec

```diff
@@ -23,8 +23,8 @@
 #
 
 Name:     liquidsoap 
-Version:  1.3.4
-Release:  3%{?dist}
+Version:  1.3.5
+Release:  0.1%{?dist}
```

Comment:
This should be: `Release:  1%{?dist}`

Reply:
I'm switching to a zero prefix in our local builds everywhere so our packages don't shadow potential distro packages.

Reply:
I could only find [this tilde discussion](https://pagure.io/packaging-committee/issue/398) in pagure. As far as I understood it was decided not to allow tilde but rather to use the Release field.

This way this works:
```
$ rpmdev-vercmp 1.3.5-0.1 1.3.5-1
1.0-0.1 < 1.0-1
```

Reply:
> I'm switching to a zero prefix in our local builds everywhere so our packages don't shadow potential distro packages.

As far as I know, there aren't any yet, as you're the one trying to bring them upstream :-)

The zero dotted prefix is intended to be used for [upstream pre-releases](https://fedoraproject.org/wiki/Packaging:Versioning#Prerelease_versions), so you might run into problems here, if you decide to ship a pre-release and/or comply with the [Fedora package versioning guidelines](https://fedoraproject.org/wiki/Packaging:Versioning).

Reply:
Given that we will be upstream on this when we upstream it to Fedora I think packaging this as a pre-release makes sense. Once I submit the package to rawhide I'll update to a 1 anyway. By using a zero dotted prefix now I can move to upstream without using an Epoch or relying on liquidsoap (or other packages) to release a new version.